sync_diff_inspector/config.toml

```diff
@@ -17,37 +17,77 @@ sample-percent = 100
 # set true if target-db and source-db all support tidb implicit column "_tidb_rowid"
 use-rowid = false
 
+# calculate the data's checksum, and compare data by checksum.
+use-checksum = true
+
 # the name of the file which saves sqls used to fix different data
 fix-sql-file = "fix.sql"
 
-# use tidb's snapshot data, sql looks like set @@tidb_snapshot="2016-10-08 16:45:26"
-source-snapshot = "2016-10-08 16:45:26"
-target-snapshot = "2016-10-08 16:45:26"
+# tables need to check.
+[[check-tables]]
+# schema name in target database.
+schema = "test"
+
+# table list which need check in target database.
+tables = ["test1", "test2", "test3"]
+
+# support regular expression, must start with '~'.
+# for example, this config will check tables with prefix 'test'.
+# tables = ["~test*"]
+# for example, this config will chek all the tables in this schema.
+# tables = ["~*"]
 
-# if check-table is nil, all tables in the schema will be checked.
+
+# schema and table in check-table must be contained in check-tables.
+# a example for comparing table with same schema and table name. 
 [[check-table]]
+# schema name.
+schema = "test"
+
 # table name.
-name = "test"
+table = "test3"
+
+# set true if comparing sharding tables with target table
+is-sharding = false
 
 # field should be the primary key, unique key or field with index. 
 # if comment this, diff will find a suitable field.
 index-field = "id"
 
-# check data's range
-# range = "age > 10 AND age < 20"
+# check data's range.
+range = "age > 10 AND age < 20"
 
-[source-db]
+# set true if comparing sharding tables with target table
+is-sharding = false
```

Comment:
repeated? this section is `table-config` or `check-table`?

Reply:
I  believe it belongs to `table-config`. According to definition in source code,`check-table` only has two fields: `schema` and `tables`.